UI/sign-in/src/App.js

```diff
@@ -0,0 +1,119 @@
+import React, { Component } from 'react';
```

Comment:
This file contains 4 components, it will get too big and hard to maintain when more code is added. I recommend breaking this file down into 4 files in the following structure:

/UI/sign-in/src/App.js [App component, import SignUp/SignIn/Forget components in this main component]
                         /Sign-Up/signup.js [SignUp component]
                        /Sign-In/signin.js [SignIn component]
                        /Froget/forget.js [Forget component]

Refer to these recommendations:
https://reactjs.org/docs/faq-structure.html
https://medium.com/styled-components/component-folder-pattern-ee42df37ec68